noir-projects/noir-protocol-circuits/crates/types/src/delayed_public_mutable/scheduled_value_change.nr

```diff
@@ -15,16 +15,55 @@ pub struct ScheduledValueChange<T> {
     pub(crate) timestamp_of_change: u64,
 }
 
+// TODO: consider lacing the context through these impls, so we know which functions are compatible with public/private/utility. For now, I've added comments to that effect.
 impl<T> ScheduledValueChange<T> {
+
+    // PUBLIC ONLY
```

Comment:
Rearranged (but didn't change) some methods, to help me see under what circumstances they're intended to be called.